TAInformation/Models/assignment.py

```diff
@@ -0,0 +1,10 @@
+from TAInformation.Models.instructor import Instructor
+from TAInformation.Models.taAssignment import taAssignment
+
+
+class ClassAssignment:
+    def __init__(self, course: str, assignment: str, TA: str):
+        self.course = course
+        self.assignment = assignment
+        self.TA = TA
+
```

Comment:
Is it necessary to have a class with no methods?  If this is supposed to be for labs, couldn't you place the data into a Labs model object?